src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Completion/RazorCompletionResolveEndpointTest.cs

```diff
@@ -236,28 +212,36 @@ public async Task Handle_Resolve_NonTagHelperCompletion_Noops()
         {
             // Arrange
             var lspDescriptionFactory = new Mock<LSPTagHelperTooltipFactory>(MockBehavior.Strict);
-            var markdown = new VSMarkupContent
+            var markdown = new MarkupContent
             {
-                Kind = VSMarkupKind.Markdown,
+                Kind = MarkupKind.Markdown,
                 Value = "Some Markdown"
             };
-            lspDescriptionFactory.Setup(factory => factory.TryCreateTooltip(It.IsAny<AggregateBoundElementDescription>(), VSMarkupKind.Markdown, out markdown))
+            lspDescriptionFactory.Setup(factory => factory.TryCreateTooltip(It.IsAny<AggregateBoundElementDescription>(), MarkupKind.Markdown, out markdown))
                 .Returns(true);
             var endpoint = new RazorCompletionResolveEndpoint(LSPTagHelperTooltipFactory, VSLSPTagHelperTooltipFactory, CompletionListCache, LoggerFactory);
-            endpoint.SetCapability(CompletionCapability, DefaultClientCapability);
+            endpoint.GetRegistration(DefaultClientCapability);
             var completionItem = new CompletionItem();
+            var parameters = ConvertToBridgedItem(completionItem);
 
             // Act
-            var newCompletionItem = await endpoint.Handle(completionItem, default);
+            var newCompletionItem = await endpoint.Handle(parameters, default);
 
             // Assert
             Assert.Null(newCompletionItem.Documentation);
         }
 
-        private CompletionList CreateLSPCompletionList(IReadOnlyList<RazorCompletionItem> razorCompletionItems)
+        private VSInternalCompletionList CreateLSPCompletionList(IReadOnlyList<RazorCompletionItem> razorCompletionItems)
         {
-            var completionList = RazorCompletionEndpoint.CreateLSPCompletionList(razorCompletionItems, CompletionListCache, supportedItemKinds: null, completionCapability: null);
+            var completionList = RazorCompletionEndpoint.CreateLSPCompletionList(razorCompletionItems, CompletionListCache, DefaultClientCapability);
             return completionList;
         }
+
+        private VSCompletionItemBridge ConvertToBridgedItem(CompletionItem completionItem)
```

Comment:
This is the one crappy thing about the "Bridge" pattern. Unfortunately I couldn't think of a better way other than creating a copy constructor in the `VSCompletionItemBridge` but there's sooooo many properties on the `VSInternalCompletionItem` + `CompletionItem` types that it'd become extremely unwieldy.